source/build/Windows/TestClientApp.vcxproj.ejs

```diff
@@ -62,7 +62,7 @@
       <WarningLevel>Level4</WarningLevel>
       <Optimization>Disabled</Optimization>
       <SDLCheck>true</SDLCheck>
-      <PreprocessorDefinitions>_DEBUG;_CONSOLE;WIN32;<%- clientDefines %>%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_DEBUG;_CONSOLE;WIN32;<% if(azureSdk){ %> <%- azureDefines %> <% } %><%- clientDefines %>%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
NIT: Use azureDefines in the way that clientDefines is used, and remove the need for azureSdk.